redis/gke-redis/cluster.tf

```diff
@@ -0,0 +1,29 @@
+# resource that defines the redis cluster
+resource "google_container_cluster" "converge_redis_cluster" {
+    name                = "${var.name}"
+    zone                = "${var.zone}"
```

Comment:
If I recall correctly `Terraform` throws a warning to have this changed to `location` since `zone` has been/will be deprecated.